File: engine-kernel/src/main/java/com/sos/scheduler/engine/kernel/cppproxy/NodeCI.java
package com.sos.scheduler.engine.kernel.cppproxy;


public interface NodeCI {
    String job_chain_path();
    String string_order_state();
    String string_next_state();
    String string_error_state();
    String string_action();
    void set_action_string(String o);
}

File: engine-kernel/src/main/scala/com/sos/scheduler/engine/kernel/cppproxy/NodeCI.scala
package com.sos.scheduler.engine.kernel.cppproxy

import com.sos.scheduler.engine.cplusplus.runtime.annotation.CppExpression

trait NodeCI {
  def job_chain_path: String
  def string_order_state: String
  def string_next_state: String
  def string_error_state: String
  def string_action: String
  def set_action_string(o: String): Unit
  @CppExpression("$->delay().millis()")
  def delay: Long
}
